src/web/services/search.service.spec.ts

```diff
@@ -179,37 +179,26 @@ describe('SearchService', () => {
     );
   });
 
-  it('should parse students into courses with sections correctly', () => {
+  it('should parse students into courses correctly', () => {
     const { students }: { students: Student[] } = mockStudents;
 
     // Number of courses should match
-    expect(coursesWithSections.length).toEqual(
+    expect(coursesWithStudents.length).toEqual(
       Array.from(new Set(students.map((s: Student) => s.courseId))).length,
     );
 
-    // Number of sections in a course should match
+    // Number of students in a course should match
     expect(
-      coursesWithSections.filter((t: SearchStudentsTable) => t.courseId === students[0].courseId)[0]
-        .sections.length,
+      coursesWithStudents.filter((t: SearchStudentsListRowTable) => t.courseId === students[0].courseId)[0]
+        .students.length,
     ).toEqual(
       Array.from(
         new Set(
           students
-            .filter((s: Student) => s.courseId === students[0].courseId)
-            .map((s: Student) => s.sectionName),
+            .filter((s: Student) => s.courseId === students[0].courseId),
         ),
       ).length,
     );
-
-    // Number of students in a section should match
-    expect(
-      coursesWithSections
-        .filter((t: SearchStudentsTable) => t.courseId === students[0].courseId)[0]
-        .sections.filter((s: StudentListSectionData) => s.sectionName === students[0].sectionName)[0]
-        .students.length,
-    ).toEqual(
-      students.filter((s: Student) => s.sectionName === students[0].sectionName).length,
-    );
```

Comment:
Same here. Should rewrite the test to check for number of students in a section instead of removing it